Lab1_Operationalizing_Pytorch_with_Mlflow/README.md

```diff
@@ -1,31 +1,127 @@
+to build end-to-end model lifecycles of Pytorch assets using Mlflow.
 
-With this 1st labs, Mlops commuunity wants to test them and provide a possible end-to-end model lifecycle of Pytorch model using Mlflow.
+## Team
 
-## Teams
+- Alexey Naiden
+- John Savage
+- Michel Vasconcelos
+- Varuna Jayasiri
 
-Teams involved in this scenario are:
+## Repo Organization
 
-- Team 1 : Michel Vasconcelos, Alexey Naiden, Varuna Jayasiri, John Savage
-- Team 2 : Itirell, Mark Peters, Carlos Leyson, Semendiak
-- Team 3 : Laszlo Sragner, yartem, Dimi, Paulo Maia
-- Team 4 : Shadab Hussain, susana brandao, Godwin Ekainu, Josh
+Following list describes the most relevant folders in which this lab is organized. Some folders in 
+this repo are self-explanatory (like [_imgs_](imgs/)) and therefore aren'r mentioned in the list.
 
+* [src](src/): Main source code folder. It comprises all software produced during this lab. _Src_ 
+itself is organized into subfolders according to which components it belongs to;
+* [docs](docs/): Binary documentation, drawings and reports produced during this lab;
 
-## Contributing
-If you want to join the initiative, please join the MLOps Community on [Slack](https://mlops-community.slack.com/join/shared_invite)
+## Proposal
+### Design Overview
+
+Our team intend to orchestrate common unix management toolkits (shell commands, ssh, etc), MLFlow
+CLI and MLFlow components to build a seamless end-to-end pipeline. It should comprehend 
+[Continuous Integration](https://en.wikipedia.org/wiki/Continuous_integration) (CI), 
+[Continuous Delivery](https://en.wikipedia.org/wiki/Continuous_delivery) (CD) and 
+[Continuous Traning](https://cloud.google.com/solutions/machine-learning/mlops-continuous-delivery-and-automation-pipelines-in-machine-learning) (CT). 
+The following figure depicts the proposed architecture.
+
+![Proposed architecture](imgs/arch.svg)
+
+**Nodes** are bare-metal or virtual machines that host a part of the pipeline process. **ML components** 
+are ML-based code that should be trained, tested and deployed by end-to-end pipeline.They must be 
+packaged as [MLFlow projects](https://mlflow.org/docs/latest/projects.html). **Training nodes**
+host the training process which runs in **Training containers**, running dockerized images built 
+with MLFlow and necessary libs to train ML components. Each ML component defines its needs, that is, 
+libs and external dependencies. To ensure traceability, provenance and model configuration management, 
+one may track hyperparameters, training steps, test results, extra data and the model itself into 
+_MLFlow Tracking Server_ hosted in the **Tracking Node**. All these items, metadata and trained models, 
+are stored in external resources: RDBMS and Cloud Storage Services. Trained models are embedded into 
+dockerized **Serving Instances**, containers running [Torchserve](https://pytorch.org/serve/) providing 
+web access to the models.
```

Comment:
minor: I would say `providing access to the models via HTTP`, because there's no web UI per se.

Reply:
Yes... pretty much better! ;-)